### DISCUSSION TITLE
Turning off selected features of DAX Studio

### DISCUSSION THREAD

**Discussion by MrKarol** [DaxStudio/DaxStudio#1300] [Q&A] [answered]
Hi All,

is it possible to turn off the following features (and how this can be done) of DAX Studio:
- software upgrade checks
- usage telemetry
- sending out error reports
- DAX Formatter

Thank you
K

**Comment by marcosqlbi** [comment, marked as answer]
They are all in Privacy options settings:
![image](https://github.com/user-attachments/assets/ce25aacf-6317-4cf1-be92-c278e52458c5)

**Comment by MrKarol** [comment]
Thank you Marco

**Reply by MrKarol** [reply]
I have follow-up question. Is it possible to package and deploy DAX Studio to users' desktops so that a user is not able to amend the privacy settings?

**Comment by marcosqlbi** [comment]
I don't know. @dgosbell is it possible?

**Comment by dgosbell** [comment]
Yes, there is an option in the installer and a command line switch that can turn off all these features. It uses a machine level registry key so only someone with admin rights can change it (usually by re-installing) see https://daxstudio.org/docs/features/privacy-settings/
